OpenIPSL/Examples/Machines/PSSE/GENROU.mo

```diff
@@ -23,37 +23,16 @@ model GENROU
     Xppq=0.2,
     R_a=0,
     Xpp=0.2,
-    H=4.28) annotation (Placement(transformation(extent={{-100,-20},{-60,20}})));
+    H=4.28) annotation (Placement(transformation(extent={{-100,-20},{
```

Comment:
I don't think we wrote it anywhere clearly (except within some internal introductory presentation). But I think these line modifications are generated because we use different line length in Dymola. We arbitrarily decided to go for 200 instead of the regular 80. 
What do you think is the best approach to use? Should I not care about that ? Should we put in the documentation to use a certain line length in the Modelica Editor? What is your input about it @dietmarw